src/services/queue/sendInitiateDataTransferMessage.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { sendInitiateDataTransferMessage } from './sendInitiateDataTransferMessage'
+import { getEnv } from '../../utils/helpers'
+import { testDataRequest } from '../../utils/tests/testDataRequest'
+import { sendSqsMessage } from './sendSqsMessage'
+jest.mock('../../utils/helpers', () => ({
+  getEnv: jest.fn()
+}))
+jest.mock('./sendSqsMessage', () => ({
+  sendSqsMessage: jest.fn()
+}))
+
+const mockGetEnv = getEnv as jest.Mock<string>
+const mockSendSqsMessage = sendSqsMessage as jest.Mock
```

Comment:
You can use the utils/tests/setup/testEnvVars for the URL, rather than mocking it

Reply:
Ah yep, hadn't noticed that! I'm sort of in two minds about centralising that kind of thing, but since it's there I may as well use it like that!